app/reducers/views/thread.js

```diff
@@ -6,160 +6,231 @@ import {FileTypes, PostTypes} from 'mattermost-redux/action_types';
 
 import {ViewTypes} from 'app/constants';
 
-function drafts(state = {}, action) {
+function currentThreadId(state = '', action) {
```

Comment:
This isn't necessary since we already have `entities.posts.selectedPostId`

Reply:
Ah gotcha, I'll make the change